backend/src/main.py

```diff
@@ -750,23 +751,57 @@ async def roll(
 
     for i in range(number):
         rolllist.append(random.randint(1, sides))
+    rolllist.sort()
 
     sumroll = sum(rolllist)
-    
+    fail = False
+
     if sumroll == sides*number:
         emoji = emojili[0]
     elif sumroll == number:
         emoji = emojili[1]
+        fail = True
     else:
         emoji = ""
 
-    message = f"🎲 **{interaction.user.name.capitalize()}** rolled a d{sides} dice {number} times: {rolllist}"
+    #SENDING GIFs
+    file = None
+    if sides == 20 and number == 1:
+        file = os.path.join(DATA_DIR, f'{DATA_DIR}/imgs/dices/roll.{random.randint(0,4)}.{sumroll-1}.webp')
+
+    if number <= 10:
+        message = f"🎲 **{interaction.user.name.capitalize()}** rolled a d{sides} dice {number} times: {rolllist}"
+    else :
+        message = f"🎲 **{interaction.user.name.capitalize()}** rolled a d{sides} dice {number} times."
+        message += f"\nMin : {min(rolllist)}"
+        message += f"\nMax : {max(rolllist)}"
+        message += f"\nMean : {sumroll/number}"
```

Comment:
maybe the message should display every result.
This could be done doing the following for a 6 sided dice (if the sided dice is greater than 100, only then it only display the 3 values):
- 20x 1
- 5x 2
- 0x 3
- 3x 4
- 6x 5
- 1x 6